spec/requests/orders_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+RSpec.describe "Orders", type: :request do
```

Comment:
Remove 1st Rspec block

Reply:
What part needs to be removed? I don't know what lines of code you mean.

Thanks for your time!